webrtc.html

```diff
@@ -853,6 +853,12 @@
                   </li>
 
                   <li>
+                    <p>If the supplied description matches the state of all
+                    tracks, clear the negotiation-needed flag.
```

Comment:
Making sure I understand:

A adds audio and video tracks to its PC. B stops video via mst.stop(), resulting in SDP with m=video 0 to its setLocalDescription, and to A's setRemoteDescription. A's video track obviously does not stop at this point, so how do we prevent a subsequent createOffer by A from including the video track again?

This seems like it needs to cause some state change on A's RtpSender.